source/2020-01-10-the-ember-times-issue-130.md

```diff
@@ -159,6 +169,8 @@ It’s also a nice place to meet up with your Ember-friends and learn about many
 
 ---
 
+> I'm excited because I think Octane is easier to teach. ~ _Jen Weber [(@jenweber)](https://github.com/jenweber)_
```

Comment:
I think the other points Jen mentions are also really useful, too. What are your thoughts on using the full quote here as well?

```suggestion
> I'm excited because I think Octane is easier to teach. That's great for both companies who hire non-Ember developers and for self-motivated beginners. With ember-auto-import in the app by default, Ember also plays nice with the broader JavaScript ecosystem. You can npm install the libraries you like, and they work without extra effort. ~ _Jen Weber [(@jenweber)](https://github.com/jenweber)_
```

Reply:
Let's then indeed use both full quotes :)